2269/2/no words

```diff
@@ -0,0 +1,4 @@
+в на о между за под по с без до для у ради к про через сквозь над перед при не ни нет никак пусть
+пускай давайте да бы б бывало вот вон давай а и именно также тоже как если где когда куда откуда
+почему лишь едва что чтобы я ты мы меня мой твой тебя он она оно они но это этот тут все там кто
+так ведь же то ну – его себя нету от из
```

Comment:
you can use more suitable name for the file. also you can move all config to `.yml` configuration file

Reply:
created config.yml and all data from bad_word_dict and no_words has been moved to this file